junction/urls.py

```diff
@@ -26,9 +27,7 @@
     url('^markdown/', include('django_markdown.urls')),
 
     # Proposals related
-    url(r'^(?P<conference_slug>[\w-]+)/proposals/', include('junction.proposals.proposal_urls')),
-    url(r'^(?P<conference_slug>[\w-]+)/proposal-comments/', include('junction.proposals.comment_urls')),
-    url(r'^(?P<conference_slug>[\w-]+)/dashboard/', 'junction.proposals.views.dashboard', name='proposal-dashboard'),
+    url(r'^(?P<conference_slug>[\w-]+)/proposal/', include('junction.proposals.urls')),
```

Comment:
This was renamed from `proposals` to `proposal`, which people noticed immediately https://twitter.com/indradhanush92/status/599982902558883842